WordPress/Sniffs/Theme/NoDeregisterCoreScriptSniff.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+/**
+ * Forbids deregistering of core scripts (jquery).
+ *
+ * @link https://make.wordpress.org/themes/handbook/review/required/#core-functionality-and-features
```

Comment:
Link is incorrect. Correct one is: https://make.wordpress.org/themes/handbook/review/required/#stylesheets-and-scripts